src/main/java/frc/trigon/robot/commands/AlignToNoteCommand.java

```diff
@@ -0,0 +1,83 @@
+package frc.trigon.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import frc.trigon.robot.RobotContainer;
+import frc.trigon.robot.commands.factories.GeneralCommands;
+import frc.trigon.robot.constants.CameraConstants;
+import frc.trigon.robot.constants.OperatorConstants;
+import frc.trigon.robot.misc.objectdetectioncamera.ObjectDetectionCamera;
+import frc.trigon.robot.subsystems.ledstrip.LEDStripCommands;
+import frc.trigon.robot.subsystems.ledstrip.LEDStripConstants;
+import frc.trigon.robot.subsystems.swerve.SwerveCommands;
+import org.trigon.hardware.RobotHardwareStats;
+import org.trigon.utilities.mirrorable.MirrorableRotation2d;
+
+import java.awt.*;
+
+public class AlignToNoteCommand extends ParallelCommandGroup {
+    private static final ObjectDetectionCamera CAMERA = CameraConstants.NOTE_DETECTION_CAMERA;
+    private static final PIDController Y_PID_CONTROLLER = RobotHardwareStats.isSimulation() ? new PIDController(0, 0, 0) : new PIDController(0, 0, 0);
```

Comment:
```suggestion
    private static final ObjectDetectionCamera CAMERA = CameraConstants.NOTE_DETECTION_CAMERA;
    private static final PIDController Y_PID_CONTROLLER = RobotHardwareStats.isSimulation() ? 
            new PIDController(0, 0, 0) : 
            new PIDController(0, 0, 0);
```
why are these two zero? calibrate the sim one for now.